core/templates/domain/learner_dashboard/learner-playlist-modal.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+    <h3 [innerHTML]="'I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_HEADER' | translate" translate-values="{sectionNameI18nId: sectionNameI18nId}"></h3>
+  </div>
+  
+  <div class="modal-body">
+    <p [innerHTML]="'I18N_LEARNER_DASHBOARD_REMOVE_ACTIVITY_MODAL_BODY' | translate" translate-values="{entityTitle: activityTitle, sectionNameI18nId: sectionNameI18nId}"></p>
+  </div>
+  
+  <div class="modal-footer">
+    <button class="btn btn-secondary" (click)="cancel()" translate="I18N_MODAL_CANCEL_BUTTON"></button>
+    <button class="btn btn-danger protractor-test-confirm-delete-interaction"
+            (click)="remove()" translate="I18N_LEARNER_DASHBOARD_REMOVE_BUTTON">
+    </button>
+  </div>
+  
```

Comment:
No newline at the end of the file? I think the overall indentation in this file is off... Can we fix that?

Reply:
added in new commit

Reply:
Github still seems to disagree... PTAL